libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -165,53 +185,58 @@ export class SkyAutonumericDirective
   }
 
   public registerOnChange(fn: (value: number | undefined) => void): void {
-    this.onChange = fn;
+    this.#onChange = fn;
   }
 
   public registerOnTouched(fn: () => void): void {
-    this.onTouched = fn;
+    this.#onTouched = fn;
   }
 
   @HostListener('blur')
   public onBlur(): void {
-    this.onTouched();
+    this.#onTouched();
   }
 
-  private getNumericValue(): number | undefined {
-    const inputValue = this.getInputValue();
-    return inputValue && !this.isInputValueTheCurrencySymbol(inputValue)
-      ? this.autonumericInstance.getNumber()
+  #getNumericValue(): number | undefined {
+    const inputValue = this.#getInputValue();
+    const result = this.#autonumericInstance.getNumber();
+    return inputValue &&
+      !this.#isInputValueTheCurrencySymbol(inputValue) &&
+      !!result
+      ? result
```

Comment:
This operation can be simplified a bit (and account for zero values being returned from the `this.#autonumericInstance.getNumber` method):
```
#getNumericValue(): number | undefined {
  const inputValue = this.#getInputValue();
  const numericValue = this.#autonumericInstance.getNumber();

  return inputValue &&
    !this.#isInputValueTheCurrencySymbol(inputValue) &&
    typeof numericValue === 'number'
    ? numericValue
    : undefined;
}
```

Reply:
Ahh... good call. done in: [modify function to allow for 0 values](https://github.com/blackbaud/skyux/pull/606/commits/95bf87efd4997ed0790ad913a9d958dd39b5713b)